test/components/user_list_item_test.js

```diff
@@ -30,4 +31,24 @@ describe("UserListItem", () => {
       expect(wrapper.text()).to.match(/treezy \(facilitator\)/i)
     })
   })
+
+  describe("user pictures", () => {
+    const user = {
+      given_name: "treezy",
+      online_at: 803,
+      is_facilitator: true,
+      picture: "http://some/image.jpg"
+    }
+
+    it("renders a list item that does have a picture", () => {
+      const wrapper = shallow(<UserListItem user={user} />)
+      expect(wrapper.find('img.picture').html())
```

Comment:
also, these tests will always pass, because there are no matcher being used with the `expect` invocation. this could easily be like `expect(wrapper.find('img.picture')).to.equal(true)`